src/Illuminate/Database/MigrationServiceProvider.php

```diff
@@ -89,7 +89,7 @@ protected function registerMigrator()
     protected function registerCreator()
     {
         $this->app->singleton('migration.creator', function ($app) {
-            return new MigrationCreator($app['files']);
+            return new MigrationCreator($app['files'], $app->basePath('stubs'));
```

Comment:
This change pretty much broke my package that relied on resolving the `MigrationCreator` (do believe my usage of this class wasn't perfect in any sense), is there a reason for the `MigrationCreator` to be bound to `migration.creator` instead of `MigrationCreator::class` which would make it resolveable from the container? @driesvints

Reply:
No idea